recipe-server/Dockerfile

```diff
@@ -13,14 +13,18 @@ RUN curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -
     apt-get update && apt-get install -y nodejs
 RUN npm install -g npm@5
```

Comment:
If we're using Yarn now, we can probably skip on installing npm5, yes?

Reply:
I thought npm was still required under the hood, but I was incorrect. I'll remove it.